src/main/java/appeng/api/features/WirelessTerminalsInternal.java

```diff
@@ -40,15 +40,18 @@ public static void installOpener() {
     private static class Opener implements WirelessTerminals.Opener {
```

Comment:
I'll merge this calss into "WirelessTerminals" directly once we merge into master. It's left-over from when api was a separate source set.